server/src/main/scala/org/http4s/server/websocket/package.scala

```diff
@@ -17,8 +17,8 @@ package object websocket {
   /**
     * Build a response which will accept an HTTP websocket upgrade request and initiate a websocket connection using the
     * supplied exchange to process and respond to websocket messages.
-    * @param read     The read side of the Exchange represents the stream of messages that should be sent to the client
-    * @param write    The write side of the Exchange is a sink to which the framework will push the websocket messages
+    * @param send     The send side of the Exchange represents the stream of messages that should be sent to the client
+    * @param receive  The receive side of the Exchange is a sink to which the framework will push the websocket messages
```

Comment:
Maybe the description of `receive` could be clearer on the line of `receiving messages from the client`

Reply:
Maybe change `client` to `consumer` (on the send parameter) and `producer` on the `receive` parameter ?

Reply:
Something like `The incoming messages to this service` works as well. In a sense, it's even easier to think about incoming and outgoing messages.

You could do that for the client as well. So you can actually say "The send side of the exchange represents the outgoing messages sent from this service."

This mostly following from @rossabaker 's incentive to make this agnostic to where its done, so it works with the client.